.github/workflows/integration-tests.yml

```diff
@@ -45,157 +45,73 @@ jobs:
             exit 1
           fi
   
-  stateless-validator:
-    name: "${{ format('{0} / {1} / {2}', matrix.el, matrix.zkvm, matrix.test) }}"
+  integration-tests:
+    name: "${{ format('{0} / {1}', matrix.zkvm, matrix.el) }}"
     strategy:
       fail-fast: false
       matrix:
-        test:
-          - execute_empty_block
-          - execute_mainnet_blocks
-          - execute_invalid_block
-          - prove_empty_block
-        zkvm: [sp1, risc0, pico, zisk, openvm, airbender]
-        el: [reth, ethrex]
         include:
+          # Stateless Validator - Reth
+          - zkvm: sp1
+            el: reth
+            tests: "execute_empty_block execute_mainnet_blocks execute_invalid_block prove_empty_block"
+            threads: 12
+          - zkvm: risc0
+            el: reth
+            tests: "execute_empty_block execute_mainnet_blocks execute_invalid_block prove_empty_block"
+            threads: 12
           - zkvm: openvm
+            el: reth
+            tests: "execute_empty_block execute_mainnet_blocks execute_invalid_block prove_empty_block"
             threads: 2
-          - threads: 12
-        exclude:
-          # ZisK
-          - zkvm: zisk
-            test: prove_empty_block # ere image intentionally doesn't bake big proving key for CI
+          - zkvm: pico
+            el: reth
+            tests: "execute_empty_block execute_invalid_block"
+            threads: 12
           - zkvm: zisk
-            test: execute_mainnet_blocks # Since v0.13.0 ZisK uses too much memory for CI runners.
-          # Ethrex
-          - el: ethrex
-            zkvm: openvm # See https://github.com/eth-act/ere/issues/168
-          - el: ethrex
-            zkvm: pico # See https://github.com/eth-act/ere/issues/174
-          - el: ethrex
-            zkvm: zisk # See https://github.com/eth-act/ere/issues/186
-          - el: ethrex
-            zkvm: airbender
-          # Pico
+            el: reth
+            tests: "execute_empty_block execute_invalid_block"
+            threads: 12
+          - zkvm: airbender
+            el: reth
+            tests: "execute_empty_block execute_mainnet_blocks execute_invalid_block"
```

Comment:
The `execute_mainnet_blocks` has been quite flaky lately, so I decided to remove it.
I want to see if we convinced now devops to give us maybe a bit more powerful machine if this can improve, or maybe the flakiness is related to Airbender SDK..